c2corg_ui/templates/waypoint/edit.html

```diff
@@ -463,7 +463,7 @@ <h2 class="heading show-phone" ><span translate>Numbers</span></h2>
             ## CUSTODIANSHIP
             <div class="form-group data half" ng-if="waypoint.waypoint_type == 'gite' || waypoint.waypoint_type == 'hut' || waypoint.waypoint_type == 'camp_site'" ng-class="{'has-success': waypoint.custodianship}">
               <label><span translate>custodianship</span> <span ng-if="waypoint.waypoint_type == 'gite' || waypoint.waypoint_type == 'camp_site' || waypoint.waypoint_type == 'hut'">*</span></label>
-              <select class="form-control" ng-options="type as mainCtrl.translate(type) for type in ${custodianship_types}" ng-model="waypoint.custodianship"><option value=""></option></select>
+              <select class="form-control" name="custodianship" ng-options="type as mainCtrl.translate(type) for type in ${custodianship_types}" ng-model="waypoint.custodianship" ng-required="waypoint.waypoint_type == 'gite' || waypoint.waypoint_type == 'hut' || waypoint.waypoint_type == 'camp_site'"><option value=""></option></select>
```

Comment:
Connection with the coords? :P 
By the way isn't there a more concise way to write ``waypoint.waypoint_type == 'gite' || waypoint.waypoint_type == 'hut' || waypoint.waypoint_type == 'camp_site'``?